File: Tareas/Marco-Oregel/Tarea1/BankTransferPayment.py
from PaymentMethod import PaymentMethod

class BankTransferPayment(PaymentMethod):

    def __init__(self,user,password,account_number,description):
        self._user = user
        self._password = password
        self._account_number = account_number
        self._description = description

    def processPayment(self,amount):
        print(f"Se ha procesado su pago de: $ {amount} pesos con transferencia bancaria desde la cuenta de: {self._user}, a la cuenta: {self._account_number}")

File: Practicas/Yahir-Hermosillo/Parcial3/Practica1/BankTransferPayment.java
class BankTransferPayment extends PaymentMethod {
    private String accountNumber;
    private String routingNumber;
    
    public BankTransferPayment(String accountNumber, String routingNumber) {
        this.accountNumber = accountNumber;
        this.routingNumber = routingNumber;
    }
    
    @Override
    public void processPayment(double amount) {
        System.out.printf("Procesando transferencia bancaria de $%.2f - Cuenta: ****%s%n",
                          amount, accountNumber.substring(accountNumber.length() - 4));
    }
}